recipes/spacy-lookups-data/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "spacy-lookups-data" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spacy_lookups_data-{{ version }}.tar.gz
```

Comment:
I saw that the release on github already has the license
you can change it to use tarballs then

Reply:
### 
```suggestion
  url: https://github.com/explosion/spacy-lookups-data/archive/v{{ version }}.tar.gz
```

Reply:
I'd rather we stuck to the release version. I've updated the manifest on Github, so for v0.0.5 I'll use that. But I do want to keep it building from pypi.